src/explorer/CosmosDBTreeItem.ts

```diff
@@ -77,8 +77,14 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
             const createdDatabase = new CosmosDBDatabase(this, connectionToAdd);
             showCreatingTreeItem(createdDatabase.label);
 
-            const appSettingToUpdate = "MONGO_URL";
             const connectionStringValue = (<string>await vscode.commands.executeCommand('cosmosDB.api.getConnectionString', connectionToAdd));
+
+            let appSettingToUpdate: string;
+            if (connectionStringValue.match(/^mongodb(\+srv)?:\/\/[^\/]*/) !== null) {
```

Comment:
In general keep regexp's as simple as they can be while still accomplishing the necessary task. To me, `/^mongo/i` is all you need to distinguish between mongo and sql. Also (nit) use `test` instead of `match` which returns `boolean` instead of `RegExpMatchArray | null`. Something like `/^mongo/i.test(connectionStringValue)`

Reply:
What is the `i` in `^/mongo/i?`?

Reply:
the letters after the '/' are flags applied to the regexp. 'i' means case-insensitive and I almost always use it

Another common one is 'g' which means 'global' which means include every match even if there are multiple